app/views/home/index.html.erb

```diff
@@ -1,39 +1,127 @@
+<style type="text/css">
+  .logo {
+    font-size: 200px !important;
+  }
+
+  .jumbotron {
+    min-height: 600px;
+  }
+
+  body {
+    padding-top: 0px;
+  }
+
+  .navbar-default {
+    background-color: rgba(44, 62, 80, 0.4);
+  }
+
+</style>
+
+
 <%= content_for :no_container do %>
   <div class="jumbotron ha-bg-parallax">
-    <div class="container white-font">
+    <div class="container white-font" style="width: 65%">
+      <h1 class="ahahamojimoji text-center logo" style="margin-bottom: -40px">ム</h1>
+
       <h1 class="text-center"><%= t :mumuki_catchphrase %></h1>
 
       <p class="text-center"><%= t :mumuki_short_description %></p>
 
-      <p class="text-center">
-        <%= link_to t(:start_practicing), categories_path, class: 'btn btn-primary special-button', role: 'button ' %>
-      </p>
+      <p class="text-center"><%= link_to t(:start_practicing), categories_path, class: 'btn btn-primary call-to-action', role: 'button ' %></p>
     </div>
   </div>
 <% end %>
 
-<div class="row">
-  <div class="col-md-4">
-    <h2 class="text-center"><%= t :mumuki_is_simple %></h2>
 
-    <p class="text-center"><%= t :mumuki_is_simple_description %></p>
-    <%= link_to t(:start_practicing), categories_path, class: 'btn btn-primary special-button btn-block', role: 'button ' %>
+<div class="mumuki-description">
+  <div class="row">
```

Comment:
Yeah, we should have a model named Pillar, so that we can edit them easily and avoid that mess.